src/lib/expansion/expansion-panel.html

```diff
@@ -1,6 +1,10 @@
 <ng-content select="mat-expansion-panel-header"></ng-content>
-<div [class.mat-expanded]="expanded" class="mat-expansion-panel-content"
-     [@bodyExpansion]="_getExpandedState()" [id]="id">
+<div class="mat-expansion-panel-content"
+     [@bodyExpansion]="_getExpandedState()"
+     (@bodyExpansion.done)="_bodyAnimation($event)"
+     (@bodyExpansion.start)="_bodyAnimation($event)"
+     [id]="id"
```

Comment:
Why are we binding the `id` here?

Reply:
That was there before my changes so I'm not sure. I _think_ it's for a11y so we can reference it somewhere else.

Reply:
I think you're right.  I think it is for `aria-owns`